config/initializers/cors.rb

```diff
@@ -17,7 +17,7 @@
 
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
-    origins "*"
+    origins "https://explorer.nervos.org"
```

Comment:
Does this affect any staging or local development environments?

Reply:
thank, I add staging, dev and local to origin. https://github.com/nervosnetwork/ckb-explorer/pull/486/commits/672f098a27417f88450a74fbcfe75a74bd1c5354